examples/illustrations/cell_site/src/cell_site-cdr.json

```diff
@@ -0,0 +1,29 @@
+[
+    {
+        "@id": "kb:9dcd8413-f681-4bc6-a66e-b70a7ecd3e4a",
+        "@type": "drafting:CapturedTelecommunicationsInformation",
+        "uco-core:hasFacet": [
+            {
+                "@type": "drafting:CapturedTelecommunicationsInformationFacet",
+                "drafting:captureCellSite": {
+                    "@id": "urn:example:cell-site-kb:cell-cite-204-16-1014-13399"
+                },
+                "observable:startTime": {
+                    "@type": "xsd:dateTime",
+                    "@value": "2021-07-29T01:23:02Z"
+                },
+                "observable:endTime": {
+                    "@type": "xsd:dateTime",
+                    "@value": "2021-07-30T02:10:50Z"
+                },
+                "drafting:partySubscription": {
+                    "@id": "kb:4b3cdcbd-6a31-462f-be9b-1ca2944c8876"
+                },
+                "drafting:partyDevice": {
+                    "@id": "kb:3fef85a7-3fb4-4170-ba4d-fc69e2de4789"
+                },
```

Comment:
I think `partyDevice` should be removed from this draft implementation.  The object is not known in this record - only some identifier string is.  (E.g. in the README, `imei TAC-SNR` and `msISDN` both look like they could be stored as identifiers, but this is outside my area of expertise.

As with discussion we've recently had on [UCO Issue 380](https://github.com/ucoProject/UCO/issues/380), it is potentially problematic to instantiate a device object and embed it as an indelible object-association in this record.  I strongly encourage removing this property from the proposal.

Reply:
`partyDevice` has been removed.